libs/plugin-analysis-api/build.gradle

```diff
@@ -26,3 +26,17 @@ tasks.named('forbiddenApisMain').configure {
 tasks.named("dependencyLicenses").configure {
   mapping from: /lucene-.*/, to: 'lucene'
 }
+
+tasks.named("thirdPartyAudit").configure {
```

Comment:
this will fail in jdk18 I think

Reply:
Yeah, it does fail with JDK < 19 :-( 

```

> Task :libs:elasticsearch-plugin-analysis-api:thirdPartyAudit FAILED
Forbidden APIs output:
==end of forbidden APIs==

> Task :server:compileJava
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use or override a deprecated API that is marked for removal.
Note: Recompile with -Xlint:removal for details.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':libs:elasticsearch-plugin-analysis-api:thirdPartyAudit'.
> All excluded classes seem to have no issues. This is sometimes an indication that the check silently failed
```

We really only want to suppress, when on JDK 19+